src/components/chips/js/contactChipsDirective.js

```diff
@@ -122,6 +124,7 @@ function MdContactChips($mdTheming, $mdUtil) {
       contactImage: '@mdContactImage',
       contactEmail: '@mdContactEmail',
       contacts: '=ngModel',
+      ngChange: "&",
```

Comment:
Why double quote here when all the other bindings have single quote?

Reply:
Opss, it just a matter of habit. In the project I'm working on we use double quotes. I'll fix.

Reply:
In the meantime I've fixed this.